src/applications/mhv/medications/containers/PrescriptionsPrintOnly.jsx

```diff
@@ -21,7 +21,13 @@ const PrescriptionsPrintOnly = () => {
   const [currentSortOption, setCurrentSortOption] = React.useState(
     selectedSortOption,
   );
-  const LIST_PAGE_PATTERN = React.useMemo(() => /^\/\d+$/, []);
+  const page = React.useMemo(
```

Comment:
Not sure, why we are not importing useEffect or useMemo instead, but I think it would be best to follow the new convention. What do you think? if you agree would you mind also importing useEffect as well. Line 25. Not a blocker. But it helps our app align with the rest of the app.